app/services/amr/n3rgy_tariffs_upserter.rb

```diff
@@ -11,7 +11,11 @@ def initialize(meter:, tariffs:, import_log:)
     def perform
       return if @tariffs.empty?
       Rails.logger.info "Upserting #{kwh_tariffs.count} tariff_prices and #{standing_charges.count} tariff_standing_charges for #{@meter.mpan_mprn} at #{@meter.school.name}"
-      TariffUpserter.new(prices_array(kwh_tariffs), standing_charges_array(standing_charges), @import_log).perform
+
+      prices_array = prices_array(kwh_tariffs)
+      standing_charges_array = standing_charges_array(standing_charges)
+
+      TariffUpserter.new(prices_array, standing_charges_array, @import_log).perform
```

Comment:
The upserter will still end up recording that its inserted 0 prices and 0 standing charges, but we'll have no record that we've basically had no tariffs from the meter. I think we should record an error to the import log if all the prices were zero, or the standing charge is zero or negative. Probably needs to be done in this class.